packages/test/functional-tests/src/test/ddsHandleEncoding.spec.ts

```diff
@@ -102,25 +110,72 @@ describe("DDS Handle Encoding", () => {
 			},
 			[handle.absolutePath] /* expectedHandles */,
 		),
+		createTestCase(
+			new SharedStringFactory(),
+			(dds) => {
+				dds.insertMarker(0, ReferenceType.Simple, { marker: handle });
```

Comment:
Any other APIs on SharedString where someone could stash a handle?

Reply:
Like `annotateMarker` or `insertText`?  I don't know anything about how it's implemented so maybe it's completely obvious that the one test case is enough. But if not, maybe good to cover a few scenarios here.  I figure you can use this one test case and just put more handles in `expectedHandles`

Reply:
They're all sort of the same; the handles would get stored in the same place. Let me think a bit about if there's anywhere else you could theoretically put a handle.